content/sensu-go/5.14/guides/clustering.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "How to run a Sensu cluster"
+linkTitle: "Running a Sensu Cluster"
+description: "Clustering is important to make Sensu more highly available, reliable, and durable. It can help you cope with the loss of a backend node, prevent data loss, and distribute the network load of agents. Read the guide to configure a Sensu cluster."
```

Comment:
```suggestion
description: "Clustering makes Sensu more available, reliable, and durable. It can help you cope with the loss of a backend node, prevent data loss, and distribute the network load of agents. Read the guide to configure a Sensu cluster."
```